CMake/external_libcurl.cmake

```diff
@@ -37,13 +44,17 @@ if(CHECK_FOR_UPDATES)
     add_definitions(-DCURL_STATICLIB) # Mandatory for building libcurl as static lib
 
     target_include_directories(curl INTERFACE $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/libcurl/libcurl_install/include>)
+        
+    target_link_libraries(curl INTERFACE debug ${CMAKE_CURRENT_BINARY_DIR}/libcurl/libcurl_install/lib/${CURL_DEBUG_TARGET_NAME}${CMAKE_STATIC_LIBRARY_SUFFIX})
+    target_link_libraries(curl INTERFACE optimized ${CMAKE_CURRENT_BINARY_DIR}/libcurl/libcurl_install/lib/${CURL_RELEASE_TARGET_NAME}${CMAKE_STATIC_LIBRARY_SUFFIX})
     
-    # libcurl require ws2_32.lib for windows only
+   # libcurl required libs per OS 
+   # (Linux require that the link dependency will be added after the libcurl link target that use it)
     if (WIN32)
-        target_link_libraries(curl INTERFACE ws2_32.lib)
+        target_link_libraries(curl INTERFACE ws2_32.lib crypt32.lib)
+    else()
+        find_package(OpenSSL REQUIRED)
```

Comment:
If that's required then it need to be propagated into the installation guide/scripts as well

Reply:
I am not sure we can explain about the online update flag when it's doing nothing for now.
The default is false and the DB is empty